specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/examples/Compute/get/ComputeInstance.json

```diff
@@ -30,6 +30,11 @@
               "adminUserName": "azureuser",
               "sshPort": 22
             },
+            "osImageMetadata": {
+              "currentImageVersion": "22.06.14",
+              "latestImageVersion": "22.07.22",
+              "isLatestOsImageVersion": false
```

Comment:
Let's drop `Os` for consistency with other properties from `isLatestOsImageVersion`

Reply:
Won't be possible as contract already changed in backend layer.